### DISCUSSION TITLE
Displaying a Widget on All Filament Admin Page

### DISCUSSION THREAD

**Discussion by lutfifirdaus** [filamentphp/filament#6751] [Help] [answered]
Hello everyone!

I would like to know, is there any way to displaying a widget on all filament admin page?

Thank you.

**Comment by pboivin** [comment, marked as answer]
Hi @lutfifirdaus, the only way I can think of would be to create your own base page classes (ListRecords, CreateRecord, EditRecord, ...). Make them extend the built-in page classes and configure the widget there. Then for all pages in your app, extend from your updated page classes.

**Reply by lutfifirdaus** [reply]
Hello @pboivin 

Yea, I think that's the only way right now to displaying widget to all pages. Thanks for the solution!
